### DISCUSSION TITLE
How to remove custom repository name from PR title ?

### DISCUSSION THREAD

**Discussion by azro352** [renovatebot/renovate#25095] [Request Help]
Hi
The usecase is with dockers on a private repository, the name of the repo is about 30-char length so the PR titles looks like 

- update customrepodockernameabitlong/dockercategory/python to 3.12 
- replace customrepodockernameabitlong/dockercategory/foo with customrepodockernameabitlong/dockerother/bar:1.2.3

Is it possible to remove the `customrepodockernameabitlong` part from PR title ?
I've seen that PR title is in fact the commit message but then I have no idea
Thanks

**Comment by rarkins** [comment]
It would need to be done using config template editing: https://docs.renovatebot.com/configuration-templates/ plus the `replace` template directive: https://docs.renovatebot.com/templates/#replace
